source/languages/cpp/generate.rb

```diff
@@ -3768,35 +3785,6 @@ def generateBlockFinder( name:"", tag_as:"", start_pattern:nil, needs_semicolon:
                 include: "#preprocessor_rule_define_line_context"
             }
         ]
-    legacy_memory_new_call = {
```

Comment:
Why was this removed?

Reply:
it wasn't used anywhere and I just realized that when I was trying to remove the `template_call.without_numbered_capture_groups`